shell/store/type-map.js

```diff
@@ -843,7 +843,7 @@ export const getters = {
 
   allTypes(state, getters, rootState, rootGetters) {
     return (product, mode = ALL) => {
-      const module = findBy(state.products, 'name', product).inStore;
+      const module = findBy(state.products, 'name', product)?.inStore;
```

Comment:
If module in undefined all the other lines are gonna give very weird output.

Could you expand on what is causing the missing product? I'm not sure that's a valid state.

If it is we probably should exit early (if everything in this function depends on it)?